pkgs/development/compilers/rust/rustfmt.nix

```diff
@@ -1,6 +1,14 @@
-{ lib, stdenv, rustPlatform, rustc, Security, asNightly ? false }:
-
-rustPlatform.buildRustPackage rec {
+{ lib
+, stdenv
```

Comment:
Please format with the new formatter

Reply:
This is soon to be done treewide anyway, no need to keep us busy with that in individual PRs unless the files where formatted previously.